docs/how-to/use-the-pebble-api.md

```diff
@@ -0,0 +1,66 @@
+# How to use the Pebble API to manage services
+
+Pebble provides a command-line interface for managing services, similar to systemd. However, Pebble distinguishes itself with its dedicated REST API.
+
+This guide demonstrates how to use the Pebble API to programmatically manage services as part of an automated workflow that starts, tests, and stops interdependent services in a CI pipeline. While shell scripts could be used for this purpose, the Pebble API offers a more robust approach with easier error handling and reusable components. This reduces manual work, errors, and ensures consistency across environments.
+
+## Use the API
+
+```{include} /reuse/api.md
+   :start-after: Start: Pebble API overview
+   :end-before: End: Pebble API overview
+```
+
+For reference information about the API, see [](../explanation/api-and-clients) and [](../reference/api).
+
+Suppose we start the Pebble daemon with no default services and an empty layer:
+
+```{terminal}
+   :input: pebble run
+2024-12-30T01:07:10.275Z [pebble] Started daemon.
+2024-12-30T01:07:10.281Z [pebble] POST /v1/services 75.042µs 400
+2024-12-30T01:07:10.281Z [pebble] Cannot start default services: no default services
+```
+
+We can then use a Python client to interact with Pebble:
+
+```python
+import ops
+
+client = ops.pebble.Client("/path/to/.pebble.socket")
+
+# get services
+services = client.get_services()
+assert len(services) == 0
+
+# add layer
+layerYAML = """
+services:
+  svc1:
+    override: replace
+    command: sleep 100
+"""
+client.add_layer(label="ci", layer=layerYAML, combine=True)
```

Comment:
`label` and `layer` are required positional args, so you'd typically use them without keywords. How about:
```suggestion
client.add_layer("label1", layerYAML, combine=True)
```